concepts/00001-enarx-sprint-framework/README.md

```diff
@@ -0,0 +1,158 @@
+
+- Backlog: for issues we want to tackle eventually, but not a high priority
+- Tracking: for issues that are high-priority, and that we want to take action on sooner rather than later. All new project issues should land here by default to ensure they don’t get lost (automatable with stock Github).
+- Active: for issues that are actively being worked on as part of the current sprint.
+- Done: when things are done, they go here (automatable with stock Github). Mostly to ensure the “Active” column stays organized.
+
+In addition to inherently categorizing our backlog, this approach allows new "categories" to be moved around or created as needed -- and, if we decide to use organization-level projects for our sprints, allows the backlog to be bisected across repositories if wanted. This will support modularization as the project grows.
+
+To start, I propose the following backlog projects:
+
+- Documentation
+- Infrastructure
+- SEV
+- SGX
+- WASM
+
+A few things to consider:
+- Downside: issues that don’t fit squarely into a category don’t have a place to go. Maybe a "Miscellaneous" project?
+- Automation opportunity: we could automatically move issues from Tracking to Active when they are assigned to someone. Automatable with Actions.
+
+### Active Sprint Board
+This board is where the team will track progress on current sprint tasks. **It should hold no backlog;** if an issue or PR is on this board, it is being actively worked on.
+
+Here’s how the columns should look:
+
+- Assigned: this indicates a task that is currently being worked on as part of the sprint. Newly-added issues land here by default (automatable with stock Github).
+- Ready for review: where pull requests land by default (automatable with stock Github); indicates it’s ready to be reviewed by someone. Pull requests should move back here from Changes Requested if a fix has been applied and it’s ready for a re-review (not in stock Github, but automatable with Actions).
+- Changes Requested: PRs move here when a review is submitted that requests changes (automatable with stock Github).
+- Approved: PRs that have been approved go here, pending merge (automatable with stock Github).
+- Done: PRs that have been merged, and their associated issues, go here (automatable with stock Github).
+
+A few things to consider:
+- Advantage: having project board be separate from backlog means we can promote issues to be part of the current sprint whenever we want to, and they can skip the backlog entirely if that’s desired.
+- Automation opportunity: If pull requests have an associated issue, don’t put the PR on the board -- instead, move the associated issue to the board (if not on it already) and treat the issue like the pull request, subject to the same movement rules as specified above. This would prevent us from having to look at the same thing twice, while still preserving a direct link to the PR in the context of its issue (since Github now displays linked PRs as a dropdown on an issue).
```

Comment:
I'm curious if this might address my question about duplicate issues between sprint / component boards. So the issue goes on the sprint board and the PR goes on the component board?